src/sudo/pipeline/edit.rs

```diff
@@ -34,16 +35,19 @@ pub fn run_edit(edit_opts: SudoEditOptions) -> Result<(), Error> {
                 &context.target_group,
             ) {
                 Ok(file) => opened_files.push((path, file)),
-                Err(error) => eprintln_ignore_io_error!("error opening {arg}: {error}"),
+                Err(error) if error.raw_os_error() == Some(40) => {
```

Comment:
Please use the respective libc constant instead.

Reply:
🤦

Reply:
I also forgot to make an issue to use the Rust native type, that's here now: #1227